src/main/java/com/daylong/songr/AlbumSongs.java

```diff
@@ -11,24 +11,52 @@
     long id;
 
     @ManyToOne
-    Album album;
+    Album songParent;
```

Comment:
This needs to match the mapped by in the Album class.
```suggestion
    Album album;
```